master/reference/calicoctl/commands/node/run.md

```diff
@@ -71,31 +79,47 @@ Description:
 
 ```
 # Start the Calico node with a specific IPv4 address for BGP.
```

Comment:
"with a specific IPv4 address for BGP."  doesn't seem relevant here since we're not using `--ip` flag in the example below. I also think this comment should be outside the code block to make it easier to read